vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher_test.go

```diff
@@ -239,17 +272,42 @@ func TestControllerFetcher(t *testing.T) {
 			expectedKey:   nil,
 			expectedError: fmt.Errorf("Cycle detected in ownership chain"),
 		},
+		{
+			key: &ControllerKeyWithAPIVersion{ControllerKey: ControllerKey{
+				Name: "test-deployment", Kind: "Deployment", Namespace: "test-namesapce"}},
+			objects: []runtime.Object{&appsv1.Deployment{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "Deployment",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-deployment",
+					Namespace: "test-namesapce",
+					// UnkownParent
```

Comment:
The parent seems to be known but not scalable?

Reply:
Maybe "not well-known" is better wording, I meant it's not in the `informersMap` map of `wellKnownController`. E.g not a Deployment, ReplicaSet, StatefulSet etc.

Reply:
Oh yeah, that makes perfect sense :+1: